jax/_src/logging_config.py

```diff
@@ -52,3 +54,4 @@ def disable_all_debug_logging():
     # difference if not other handlers are attached, but set it back in case
     # something else gets attached (e.g. absl logger) and for consistency.
     logger.setLevel(logging.WARNING)
```

Comment:
Hmm this logic is no longer valid right? It's kind of an edge case, but if JAX_LOGGING_LEVEL is set and then the debug level is changed, it will reset JAX_LOGGING_LEVEL potentially. Maybe just add a TODO to fix this for now.